argumentsExt/Annotation/StepInjectorArgument.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the StepArgumentInjectorBehatExtension project.
+ *
+ * (c) Rodrigue Villetard <rodrigue.villetard@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Gorghoa\StepArgumentInjectorBehatExtension\Annotation;
+
+/**
+ * @author Rodrigue Villetard <rodrigue.villetard@gmail.com>
+ */
+interface StepInjectorArgument
+{
+    /**
+     * @return string
+     */
+    public function getName();
+
+    /**
+     * @return string
+     */
+    public function getArgument();
+}
```

Comment:
This interface may be still too related with the state sharing business 🤔 

Why on earth should we force the annotation to deal with the `getName` ?

`getArgument` may be enough